CONTRIBUTING.md

```diff
@@ -68,19 +68,15 @@ Before you get started, you need to set yourself up with an environment in which
 
 ### JDK
 
-1. If you are working with **Windows** or **Linux**, install one of the following [Java Development Kits](http://www.oracle.com/technetwork/java/javase/downloads/index.html) on your local drive. If you are working on a **Mac**, simply confirm that the JDK shipped with your Mac OS X is *Java SE Development Kit 7 (also known as Java SE 7u80)* or newer using the command `java -version`, then skip to step 4 below: 
+1. If you are working with **Windows** or **Linux**, install one of the following [Java Development Kits](http://www.oracle.com/technetwork/java/javase/downloads/index.html) on your local drive. If you are working on a **Mac**, simply confirm that the JDK shipped with your Mac OS X is *Java SE Development Kit 8 (also known as Java SE 8u152)* or newer using the command `java -version`, then skip to step 4 below: 
```

Comment:
The sentence about "Windows or Linux" is confusing, pointing to a link where JDK 8 & 9 are available, it should be clear that we support only JDK 8